drivers/bitcoiva.js

```diff
@@ -0,0 +1,36 @@
+const Driver = require('../models/driver');
+const request = require('../lib/request');
+const Ticker = require('../models/ticker');
+const { parseToFloat } = require('../lib/utils.js');
+
+/**
+ * @memberof Driver
+ * @augments Driver
+ */
+class Bitcoiva extends Driver {
+  /**
+   * @augments Driver.fetchTickers
+   * @returns {Promise.Array<Ticker>} Returns a promise of an array with tickers.
+   */
+  async fetchTickers() {
+    const { pairlist: markets } = await request('https://www.bitcoiva.com/api/tickers');
+
+    return Object.keys(markets).map((market) => {
+      const ticker = markets[market];
+      // eslint-disable-next-line no-useless-escape
+      const [base, quote] = market.split('\/');
+
+      return new Ticker({
+        base,
```

Comment:
Always nice to have the names for making listing a lot easier :)
```suggestion
        base,
        baseName: ticker. base_full_name,
```